tech-far-hub/src/components/summary.tsx

```diff
@@ -1,12 +1,13 @@
 import * as React from "react";
-import { SummaryBox, SummaryBoxContent, SummaryBoxHeading } from "@trussworks/react-uswds";
 
 export const Summary = ({ children, heading }: { children: React.ReactNode; heading?: string }): JSX.Element => {
   heading = heading ? heading : "Summary";
   return (
-    <SummaryBox>
-      <SummaryBoxHeading headingLevel="h3">{heading}</SummaryBoxHeading>
-      <SummaryBoxContent>{children}</SummaryBoxContent>
-    </SummaryBox>
+    <div className="usa-alert usa-alert--info">
```

Comment:
Looking back, it's probably better to not use Trussworks, unless you really need to. One less dependency. AND, USWDS is moving faster than Trusswork in terms of releases.